idd/Energy+.idd.in

```diff
@@ -18632,6 +18632,28 @@ SurfaceProperty:SolarIncidentInside,
        \type object-list
        \object-list ScheduleNames
 
+SurfaceProperty:IncidentSolarMultiplier,
+      \min-fields 3
+  A1, \field Surface Name
+      \required-field
+      \type object-list
+      \object-list SurfaceNames
+      \note Enter the name of an exterior window outside surface object
+  N1, \field Incident Solar Multiplier
+      \note a constant multiplier for window solar transmittance
+      \note and visible transmittance. If the Shading Multiplier Schedule Name is
+      \note defined, the product of these two will be the final shading multiplier.
+      \type real
+      \units dimensionless
+      \minimum 0
+      \maximum 1
```

Comment:
@EnergyArchmage you had suggested we could go higher than 1.0 for some interesting simulation studies.  Should this be > 0?  I haven't seen whether the schedule in the next field allows for > 1 or not, but the comment suggests it should be 0 < x < 1.

Reply:
@EnergyArchmage @Myoldmopar An interesting and constructive comment on the multiplier value being greater than 1 to allow other use cases. Indeed it is possible but it stretches the original design purpose which is to simulate a simplified shading effect for windows. I suggest we keep it as it is and come back in future to expand it for other use cases including Brent's mentioned case, shading for opaque walls or special dynamic coating for surfaces etc.

Reply:
I don't recall that multiplier = 0.0 would be a problem in the code.   Could be useful for elimination parametric studies.

Reply:
0 and 1 should both be inclusive. 0 corresponds to a full-shading case and 1 corresponds to no shading.

Reply:
multiplier = 0 should be allowed to model special cases such as 'opaque' windows or windows covered by aluminum foil.

Reply:
@Myoldmopar I didn't find the comment suggesting the incident solar multiplier greater than 0 (not inclusive). Could you please point me there? I'll change it to inclusive.